### DISCUSSION TITLE
Dataset restore after firmware upgrade?

### DISCUSSION THREAD

**Discussion by lalley2ka** [Mr-MIBonk/M.I.B._More-Incredible-Bash#546] [Q&A]
I have release train MHI2_US_POG11_P5159 installed in a Porsche Cayenne with a Bose sound system.  Can I safely upgrade to MHI2_US_POG11_K5186 firmware that is one the mibsolution.one website?  Do I risk losing the datasets that control the amp?  If so, can I restore them using the restore storage.raw function?

**Comment by Tomt97** [comment]
following.,  I dont know how to restore data set  (audio-wise) for my Audi

**Comment by lalley2ka** [comment]
I really don't know for sure, having never done it.  But I would think the M.I.B.  backup_restore -> restore storage .raw datasets from backup function would be worth a try, assuming you did the proper backup functions when you first installed the G.E.M.

**Reply by Tomt97** [reply]
so i do see restore .raw  now how do we edit the .raw file?
